### PR TITLE
Fix replace story video

### PR DESCRIPTION
**Pull request by iamr0s**
Fix replace story video by replace uri.
old: hook onCreate when start video activity.
new: change "bilibili://story/" to "bilibili://video/" when get the url.

### CODE REVIEW

**Review by yujincheng08 — APPROVED** [review]

**Comment by iamr0s** [comment]
sorry, I misspelled a word

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: Android CI
 on:
   push:
     branches: [ master ]
+  workflow_dispatch:
```
Comment (yujincheng08) [inline .github/workflows/android.yml:6]: ?

Comment (yujincheng08) [inline .github/workflows/android.yml:6]: ?

**Review by yujincheng08 — CHANGES_REQUESTED** [review]

File: .github/workflows/android.yml (hunk 1 above)
Comment (iamr0s) [reply .github/workflows/android.yml:6]: I try to test in my actions.^_^
image

Comment (yujincheng08) [reply .github/workflows/android.yml:6]: dont submit it to upstream.

Comment (iamr0s) [reply .github/workflows/android.yml:6]: Ok, sorry

**Review by yujincheng08 — APPROVED** [review]

**Comment by zjns** [comment]
~~没有用到的 import 没删！~~

**Pull request merged**